pytest/testsuites/test_lis.py

```diff
@@ -0,0 +1,19 @@
+"""Runs 'LIS-Tests.xml' using Pytest."""
+import conftest
+import pytest
+from node_plugin import Node
+
+
+@pytest.mark.lisa(
+    platform="Azure", category="Functional", area="LIS_DEPLOY", tags=["lis"], priority=0
```

Comment:
To avoid repeat values, and write overview description for a suite of test cases. It needs a metadata to suite level. the area should be same for cases of a suite, and category, tags can be overwritten on case leve.

Reply:
Yes, marks can be applied at multiple scopes. You could instead mark the whole module, or just a class within a module.